verticapy/jupyter/extensions/chart_magic.py

```diff
@@ -192,30 +192,427 @@ def chart_magic(
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_2.html
+
+    .. code-block:: python
+
+        %%chart -k pearson
+        SELECT
+            *
+        FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k pearson
+        SELECT * FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        heatmap = _
+        heatmap.write_html("figures/jupyter_extensions_chart_magic_chart_magic_3.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_3.html
+
+    .. code-block:: python
+
+        %%chart -k hist
+        SELECT
+            pclass,
+            SUM(survived)
+        FROM titanic GROUP BY 1;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k hist
+        SELECT
+            pclass,
+            SUM(survived)
+        FROM titanic GROUP BY 1;
+
+    .. ipython:: python
+        :suppress:
+
+        hist = _
+        hist.write_html("figures/jupyter_extensions_chart_magic_chart_magic_4.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_4.html
+
+    .. code-block:: python
+
+        %%chart -k scatter
+        SELECT
+            PetalLengthCm,
+            PetalWidthCm,
+            Species
+        FROM iris;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k scatter
+        SELECT
+            PetalLengthCm,
+            PetalWidthCm,
+            Species
+        FROM iris;
+
+    .. ipython:: python
+        :suppress:
+
+        scatter = _
+        scatter.write_html("figures/jupyter_extensions_chart_magic_chart_magic_5.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_5.html
+
+    .. code-block:: python
+
+        %%chart -k boxplot
+        SELECT * FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k boxplot
+        SELECT * FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        boxplot = _
+        boxplot.write_html("figures/jupyter_extensions_chart_magic_chart_magic_6.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_6.html
+
+    Exporting to HTML
+    =================
+    Export a chart to HTML:
+
+    .. code-block:: python
+
+        %%chart -k scatter -o "my_graphic"
+        SELECT * FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        %%chart -k spearman
+        SELECT * FROM titanic;
+
+    .. ipython:: python
+        :suppress:
+
+        export = _
+        export.write_html("figures/jupyter_extensions_chart_magic_chart_magic_7.html")
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_chart_magic_chart_magic_7.html
+
+    .. code-block:: python
+
+        file = open("my_graphic.html", "r")
```

Comment:
will it display the result?

Reply:
in this case, just write a sentence. "The following lines will open the HTML file."

.. note:: The HTML graphic can be embedded in an external environment, such as a website.